src/datasets/heartbeat.md

```diff
@@ -0,0 +1,89 @@
+# Accessing Heartbeat data
+
+[Heartbeat][heartbeat] survey studies return telemetry on user engagement with the survey prompt.
+The telemetry is received using the `heartbeat` document type,
+which is [described in the Firefox source tree docs][hbping].
+
+These pings are aggregated into the `telemetry_heartbeat_parquet` table,
+and may also be accessed using the Dataset API.
+
+## Linking Heartbeat responses to telemetry
+
+Heartbeat responses may be linked to Firefox telemetry
+if there is a `"includeTelemetryUUID": true` key in the `arguments` object
+of the [`show-heartbeat` recipe][show-heartbeat].
+
+Heartbeat never reports telemetry `client_id`s to SurveyGizmo, but,
+when `includeTelemetryUUID` is true,
+a new, unique user ID is generated and reported to SurveyGizmo
+as the `userid` URL variable.
+Simultaneously, a `heartbeat` ping is sent to Mozilla,
+containing both the `client_id` and the new `userid`.
+
+The `userid` is reported by appending it to the `surveyId` field of the ping, like:
+
+```
+hb-example-slug::e87bcae5-bb63-4829-822a-85ba41ee5d53
+```
+
+These can be extracted from the Parquet table for analysis using expressions like:
+
+```
+SPLIT(payload.survey_id,'::')[1] AS Shield_ID
```

Comment:
I'm confused by this. Is the `client_id` reported in `telemetry_heartbeat_parquet` not a telemetry `client_id`? If not, we should call this out explicitly in the introduction of the article. 

Otherwise, please clarify when one would want to extract the `shield_id`. I can't think of a reason offhand.

Reply:
I should streamline the nomenclature -- `client_id` is a telemetry client_id; `userid`/`Shield_ID` is the random token given to SurveyGizmo. Knowing pairs of (`client_id`, `surveygizmo_userid`) lets you join survey responses to telemetry. I'll take another swing at it.

Reply:
Ah! Are the survey responses stored somewhere else? If so, I think that's the piece I was missing.

Reply:
They are! I'll make sure that's clear.